HEC.FDA.View/Study/StudyView.xaml.cs

```diff
@@ -92,7 +92,7 @@ private void UserControl_Loaded(object sender, RoutedEventArgs e)
 
         private void QuickStartGuide_Click(object sender, RoutedEventArgs e)
         {
-            string quickStartLink = "https://www.hec.usace.army.mil/confluence/display/FDAQSG/HEC-FDA+Quick+Start+Guide";
+            string quickStartLink = "https://www.hec.usace.army.mil/confluence/fdadocs/fdaqsg/hec-fda-quick-start-guides-152636942.html";
```

Comment:
this should not be how you do this. use a FWLINK ID so you can maintain and never get page not found.